ui/Screen/UserRegistration.svelte

```diff
@@ -11,7 +11,7 @@
 
   let step = 1;
   // TODO(merle): Get actual user profile (id, name, imageUrl, avatarFallback)
-  let handle = "cloudhead";
+  let handle = "";
```

Comment:
Why did you remove the handle? This is the identity name aka default handle that automatically fills the user handle registration input field. One of the tests is failing due to this.

Reply:
How is this related? The tests use `nope` as far as I went through the. It is a weird default to have when opening the screen and was in the way when testing it in the running app.

Reply:
There are two different tests. `nope` is used to test for a handle that is already taken ("second test"). The first test is about navigating through the modal. The modal should be pre-filled with your chosen identity name, which will come from the storage in the future, for now it is hardcoded.